include/seahorn/Expr/Smt/Z3.hh

```diff
@@ -587,6 +588,14 @@ public:
       *(out++) = z3.toExpr(r[i]);
   }
 
+  /// Dumps assertions currently in the solver to stderr
+  void dump() const {
+    z3::ast_vector r(ctx, Z3_solver_get_assertions(ctx, solver));
+    ctx.check_error();
+    for (unsigned i = 0; i < r.size(); ++i)
+      std::cerr << *(z3.toExpr(r[i])) << std::endl;
```

Comment:
`llvm::errs()` instead of `std::errs()`

If this is the only reason for including `ExprCore.hh` then move it to a cc file

Reply:
removing this code for now from this PR.